src/modules/Routing.rb

```diff
@@ -50,8 +50,9 @@ class RoutingClass < Module
     # "routes" file location
     ROUTES_FILE = "/etc/sysconfig/network/routes"
 
-    SYSCTL_IPV4_PATH = ".etc.sysctl_conf.\"net.ipv4.ip_forward\""
-    SYSCTL_IPV6_PATH = ".etc.sysctl_conf.\"net.ipv6.conf.all.forwarding\""
+    SYSCTL_AGENT_PATH = ".etc.sysctl_conf"
+    SYSCTL_IPV4_PATH = "net.ipv4.ip_forward"
+    SYSCTL_IPV6_PATH = "net.ipv6.conf.all.forwarding"
```

Comment:
Oh, I see, `SYSCTL_IPV9_PATH` are used as an argument to `sysctl -w`. Then I'd use

```rb
# sysctl keys, used as *single* SCR path components below
IPV4_SYSCTL = "net.ipv4.ip_forward"
IPV6_SYSCTL = "net.ipv6.conf.all.forwarding"
# SCR paths
SYSCTL_AGENT_PATH = path(".etc.sysctl_conf")
SYSCTL_IPV4_PATH = SYSCTL_AGENT_PATH + IPV4_SYSCTL
SYSCTL_IPV6_PATH = SYSCTL_AGENT_PATH + IPV6_SYSCTL
```